app/modules/Documentation/services/Article.php

```diff
@@ -0,0 +1,345 @@
+<?php
+
+/**
+ * This file is part of Vegas package
+ *
+ * @author Tomasz Borodziuk <tomasz.borodziuk@amsterdam-standard.nl>
+ * @copyright Amsterdam Standard Sp. Z o.o.
+ * @homenavigation https://bitbucket.org/amsdard/vegas-phalcon
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Documentation\Services;
+
+use Documentation\Models\Article as ArticleModel;
+use Documentation\Models\Version as VersionModel;
+use Documentation\Models\Category as CategoryModel;
+
+class Article {    
+    public function updateContent($id, $content, $contentRendered, $archival=false)
+    {
+        $article = $this->getArticle($id);
+        if(!empty($article)) {
+            $article->content = $content;
+            $article->contentRendered = $contentRendered;
+            if($archival) return $article->makeCopyToArchive(0);
+            return $article->save();
+        }    
+        return false;
+    }
+    
+    public function updatePosition($id, $position)
+    {
+        $article = $this->getArticle($id);
+        if(!empty($article)) {
+            $article->position = $position;
+            return $article->save();
+        }
+        return false;
+    }
+    
+    public function getArticle($id)
```

Comment:
findById accepts string or MongoId. In post and routing params id is saved as string, so why should be here MongoId?
```php 
    /**
         * Find a document by its id (_id)
         *
         * @param string|\MongoId $id
         * @return \Phalcon\Mvc\Collection
         */
        public static function findById($id)
        {
        }
```